src/ros_can_nodes/CMakeLists.txt

```diff
@@ -85,11 +87,14 @@ find_package(Boost REQUIRED)
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-    INCLUDE_DIRS include
-    CATKIN_DEPENDS std_msgs ros_type_introspection topic_tools owr_messages
-    DEPENDS Boost
+#  INCLUDE_DIRS include
+#  CATKIN_DEPENDS ros_type_introspection
+#  DEPENDS Boost
```

Comment:
commented code

Reply:
dont think there is (or will be) any packages that depend on ros_can_nodes. the call should be empty if thats the case (even though the call itself is mandatory). a new ros package has these lines commented